Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/AlpideCoder.h

```diff
@@ -99,10 +101,13 @@ class AlpideCoder
   static constexpr int EOFFlag = -100; // flag for EOF in reading
 
   AlpideCoder() = default;
-  ~AlpideCoder() = default;
+  ~AlpideCoder() { delete mNoisyPixels; }
```

Comment:
I think it is better to avoid deleting the map: usually it will be set from ccdb object, which is cached.

Reply:
@shahor02  Right.  But currently, we swap this map from a root file.  For the moment, I would delete it, just to avoid a trivial memory leak.   Of course, once we get to a ccdb implementation, we will have to do something else here.

Reply:
Are working with multiple noise maps within single session? If not, then I would not care about the leak.
BTW, when you delete the object pointed by a static pointer data member, the pointer must be set to 0.

Reply:
@shahor02 OK. No problem. Let's keep this object un-deleted.